MTDatesTests/MTDatesTests.m

```diff
@@ -896,7 +901,18 @@ - (void)test_hoursUntilDate
     XCTAssertEqual([date2 mt_hoursUntilDate:date], 4);
 }
 
+#pragma mark - SECONDS
 
+- (void)test_oneSecondNext {
+    NSDate *date = [_militaryTimeFormatter dateFromString:@"12/18/1992 5:45:00"];
+    NSDate *date2 = [_militaryTimeFormatter dateFromString:@"12/18/1992 5:45:01"];
+    
+    // In case the formatter's dateFormat changes
+    XCTAssertNotNil(date);
+    XCTAssertNotNil(date2);
+    
+    XCTAssertEqualObjects(date.mt_oneSecondNext, date2);
+}
```

Comment:
Crap, I should have spotted that one. Good call. Copy + Paste is never reliable ;)